lib/task-data/tasks/bootstrap-ubuntu.js

```diff
@@ -7,12 +7,12 @@ module.exports = {
     injectableName: 'Task.Linux.Bootstrap.Ubuntu',
     implementsTask: 'Task.Base.Linux.Bootstrap',
     options: {
-        kernelFile: 'vmlinuz-3.16.0-25-generic',
-        initrdFile: 'initrd.img-3.16.0-25-generic',
+        kernelFile: 'vmlinuz-3.19.0-56-generic',
+        initrdFile: 'initrd.img-3.19.0-56-generic',
         kernelUri: '{{ api.server }}/common/{{ options.kernelFile }}',
         initrdUri: '{{ api.server }}/common/{{ options.initrdFile }}',
-        basefs: 'common/base.trusty.3.16.0-25-generic.squashfs.img',
-        overlayfs: 'common/discovery.overlay.cpio.gz',
+        basefs: 'common/base.trusty.3.19.0-56-generic.squashfs.img',
+        overlayfs: 'common/discovery.3.19.0-56-generic.overlay.cpio.gz',
```

Comment:
@benbp as long as we're rendering option values why not pass each full filename and path in as an option?  What if I want to boot my own custom overlay (or initrd, kernel, etc) called `my.custom.overlay.version.1.cpio.gz` put in `my_overlays/` instead of `/common`;  my graph would pass `overlayfs` and rendered as:
`overlayfs: {{option.overlayfs}}` = `my_overlays/my.custom.overlay.version.1.cpio.gz`.

Are we intentionally enforcing a file naming convention?  Just throwing it out there.